frontend/src/modules/home/onboarding/footer.tsx

```diff
@@ -7,16 +7,18 @@ import type { Organization } from '~/types';
 import { ArrowLeft, Redo } from 'lucide-react';
 import { useNavigate } from '@tanstack/react-router';
 import { toast } from 'sonner';
+import { useTranslation } from 'react-i18next';
 
 const StepperFooter = ({ organization }: { organization?: Organization | null }) => {
   const navigate = useNavigate();
+  const { t } = useTranslation();
   const { nextStep, prevStep, isLastStep, isOptionalStep, hasCompletedAllSteps, activeStep } = useStepper();
 
   useEffect(() => {
-    if (activeStep === 0 || !hasCompletedAllSteps) return
-    toast.success('You are good to go!');
+    if (activeStep === 0 || !hasCompletedAllSteps) return;
+    toast.success(t('common:onboarding_stepperFooter_success'));
```

Comment:
please read readme in /locales. the translation key is not consistent with other success messages